sci-physics/fastjet-contrib/fastjet-contrib-9999.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit subversion toolchain-funcs
+
+MY_PN=fjcontrib
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="3rd party extensions of FastJet."
+HOMEPAGE="https://fastjet.hepforge.org/contrib/"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS=""
+
+DEPEND=">=sci-physics/fastjet-3.4.1"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.049-ar.patch
+	"${FILESDIR}"/${PN}-1.055-ar-part2.patch # https://github.com/fjcontrib/fjcontrib/issues/3
+)
+
+src_unpack() {
+	export ESVN_REPO_URI="https://svn.hepforge.org/fastjetsvn/contrib/trunk"
```

Comment:
Shouldn't this be in global scope?

Reply:
I'll move it, since it does not matter. I copied it from another ebuild where I had multiple svns.